pmoves/services/session-context-worker/main.py

```
@@ -79,60 +56,70 @@
+
+def _nats_loop_done(task: asyncio.Task) -> None:
+    """Callback for NATS resilience loop task completion."""
+    if not task.cancelled():
+        exc = task.exception()
+        if exc:
+            logger.error("NATS resilience loop crashed unexpectedly: %s", exc, exc_info=True)
+
+
 # FastAPI app for health endpoint
 @asynccontextmanager
 async def lifespan(app: FastAPI):
```

Ruff (0.14.10)

Unused function argument: `app`

(ARG001)